packages/nodes-base/nodes/Slack/V2/SlackV2.node.ts

```diff
@@ -710,10 +710,10 @@ export class SlackV2 implements INodeType {
 							qs.inclusive = filters.inclusive as boolean;
 						}
 						if (filters.latest) {
-							qs.latest = new Date(filters.latest as string).getTime() / 1000;
+							qs.latest = filters.latest as string;
 						}
 						if (filters.oldest) {
-							qs.oldest = new Date(filters.oldest as string).getTime() / 1000;
+							qs.oldest = filters.oldest as string;
 						}
```

Comment:
this is breaking change, we need to add a new node version
I would suggest to add simple helper
```
						const processTimestampParameter = (nodeVersion: number, timestamp: string) => {
							if (nodeVersion < 2.4) {
								return new Date(timestamp).getTime() / 1000;
							}

							return timestamp;
						};
```